### PR TITLE
feat: add Danish localization

### PR DESCRIPTION
**Pull request by ddfreiling**
Finally got around to adding Danish localization for the Ace app too.

### CODE REVIEW

**Comment by ddfreiling** [comment]
@MyDK care to review this one as well? Tak :)

File: src/shared/l10n/locales/da.json [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            "a11y": "Tilgængelighed",
+            "allPresent": "Alle krævede metadata felter om tilgængelighed er til stede.",
+            "missing": "Manglende tilgængeligheds Metadata:",
+            "name": "Navn",
+            "value": "Værdi"
+        },
+        "outlines": "Struktur",
+        "outlinesSection": {
+            "headings": "Overskrift struktur",
+            "html": "HTML struktur",
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:118]:
```suggestion
            "html": "HTML-struktur",
```

File: src/shared/l10n/locales/da.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        "metadataSection": {
+            "a11y": "Tilgængelighed",
+            "allPresent": "Alle krævede metadata felter om tilgængelighed er til stede.",
+            "missing": "Manglende tilgængeligheds Metadata:",
+            "name": "Navn",
+            "value": "Værdi"
+        },
+        "outlines": "Struktur",
+        "outlinesSection": {
+            "headings": "Overskrift struktur",
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:117]:
```suggestion
            "headings": "Overskriftstruktur",
```

File: src/shared/l10n/locales/da.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        "history": "Historik",
+        "maximize": "Maksimér Sidebar",
+        "minimize": "Minimér Sidebar",
+        "reRun": "Kør igen",
+        "settings": "Indstillinger"
+    },
+    "splash": {
+        "browseForFileOrFolder": "søg efter en __fil__ eller en __mappe__ .",
+        "clickToBrowse": "klik her for at gennemse.",
+        "dropHere": "Træk en EPUB fil eller mappe ind her,",
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:157]:
```suggestion
        "dropHere": "Træk en EPUB-fil eller -mappe ind her",
```

File: src/shared/l10n/locales/da.json [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+        "or": "eller",
+        "orDropSidebar": "eller på __ knappen i sidebaren,",
+        "title": "Ace, af DAISY"
+    },
+    "tablePagination": {
+        "firstPage": "Første Side",
+        "lastPage": "Sidste Side",
+        "nextPage": "Næste Side",
+        "previousPage": "Forrige Side"
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:166]: @ddfreiling lines 163,164 and165 ends with a comma, then shouldn't line 165 too?

File: src/shared/l10n/locales/da.json [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+        "summary": "Oversigt",
+        "summarySection": {
+            "best-practice": "Best Practice",
+            "critical": "Kritisk",
+            "minor": "Mindre",
+            "moderate": "Moderat",
+            "other": "Andre",
+            "serious": "Stor",
+            "total": "Total",
+            "type": "Type"
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:130]: @ddfreiling  - should this line end with at comma?

File: src/shared/l10n/locales/da.json [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+        "save": "Gem",
+        "title": "Ace Indstillinger",
+        "userInterfaceLanguage": "Sprog",
+        "userInterfaceLanguageTip": "(vælg sprog til grænsefladens tekst og beskeder)"
+    },
+    "report": {
+        "images": "Billeder",
+        "imagesSection": {
+            "NA": "N/A",
+            "altAttribute": "'alt' attribut",
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:99]:
```suggestion
            "altAttribute": "'alt'-attribut",
```

File: src/shared/l10n/locales/da.json [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+        "savereport": "Gem Rapport",
+        "select": "Vælg",
+        "ziparchive": "Zip Arkiv"
+    },
+    "enhancedTable": {
+        "filterBy": "Filtrér efter",
+        "rowsPerPage": "Rækker per side:",
+        "sortBy": "Sorter efter: "
+    },
+    "kbgoonline": "ÅBEN ONLINE",
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:20]:
```suggestion
    "kbgoonline": "ÅBN ONLINE",
```

File: src/shared/l10n/locales/da.json [hunk 8]
```diff
@@ -0,0 +1,174 @@
+{
+    "dialog": {
+        "allfiles": "Alle filer",
+        "check": "Tjek",
+        "choosedir": "Vælg en mappe",
+        "chooseepub": "Vælg en EPUB fil eller mappe",
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:6]:
```suggestion
        "chooseepub": "Vælg en EPUB-fil eller -mappe",
```

File: src/shared/l10n/locales/da.json [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+        "chooseepubdir": "Vælg en EPUB mappe",
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:7]:
```suggestion
        "chooseepubdir": "Vælg en EPUB-mappe",
```

File: src/shared/l10n/locales/da.json [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+        "chooseepubfile": "Vælg en EPUB fil",
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:8]:
```suggestion
        "chooseepubfile": "Vælg en EPUB-fil",
```

File: src/shared/l10n/locales/da.json [hunk 11]
```diff
@@ -0,0 +1,174 @@
...
+        "choosereport": "Vælg en Ace rapport",
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:9]:
```suggestion
        "choosereport": "Vælg en Ace-rapport",
```

File: src/shared/l10n/locales/da.json [hunk 12]
```diff
@@ -0,0 +1,174 @@
...
+        "open": "Åben",
+        "savereport": "Gem Rapport",
+        "select": "Vælg",
+        "ziparchive": "Zip Arkiv"
```
Comment (MyDK) [inline src/shared/l10n/locales/da.json:13]:
```suggestion
        "ziparchive": "Zip-arkiv"
```

File: src/shared/l10n/locales/da.json (hunk 4 above)
Comment (ddfreiling) [reply src/shared/l10n/locales/da.json:166]: Not in JSON, comma is only between adjacent properites.

**Comment by ddfreiling** [comment]
@MyDK Thanks for review. I've updated as suggested. I have kept casing and started sentences with lower-case where it did so in the English version. It makes sense if you run the app and see the context of the text.
The `__` stuff is placeholder or hints to the app to bold text, so I kept that as well.

**Comment by ddfreiling** [comment]
@danielweck should be ready now.

**Comment by danielweck** [comment]
Noted!

**Pull request merged**